Mobile/ShopLens/Droid/IO/ShopLensPictureDirectoryCreator.cs

```diff
@@ -14,11 +14,7 @@ private void CreateDirectoryForPictures(File pictureDirectory)
         {
             if (!pictureDirectory.Exists())
```

Comment:
Right here we're checking whether a said directory exists or not, if it doesn't we create the directory.